src/IOManagement.cpp

```diff
@@ -0,0 +1,172 @@
+#include "IOManagement.h"
+#include <array>
+
+// TODO: PID, TimeOut, Thermistor, pwm
+
+// Solar array voltage, current, and PWM pins (controlled by PID) and storage variable
+volatile ArrayData arrayData[NUM_ARRAYS];
+
+struct ArrayPins {
+    uint8_t voltPin;
+    INA281Driver currPin;      // need to rewrite
+    double outputPWM;
```

Comment:
I know I said to put these two doubles in the struct, but the way we currently have it causes compile errors at lines 19-21. We might have to store these in separate arrays like we had originally.